src/content/language/constructors.md

```diff
@@ -51,12 +51,12 @@ To instantiate a class, use a generative constructor.
 <?code-excerpt "point_alt.dart (idiomatic-constructor)" plaster="none"?>
```

Comment:
Hey @danielerat, thanks for your PR. Our code-excerpts correspond to Dart files in the `/examples` folder of this repo; this one corresponds to [this "docregion" in this file](https://github.com/dart-lang/site-www/blob/main/examples/misc/lib/language_tour/classes/point_alt.dart#L7). Could you update that to match your changes as well? And then run  `./dash_site refresh-excerpts` in your terminal (instructions in the site readme [here](https://github.com/dart-lang/site-www?tab=readme-ov-file#refresh-code-excerpts)). Thanks!

Reply:
Yes sure.